starter_code/views/randomBeer.hbs

```diff
@@ -0,0 +1,3 @@
+<h1>Random Beer</h1>
+
+{{> _singleBeer theBeer}}
```

Comment:
the partial name should be the same as the name of the partial folder: _singleBeer should be singleBeer